website_partner_contact/README.rst

```diff
@@ -0,0 +1,31 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner first name and last name for website_sale
+================================
+
+This module was written to extend the functionality of contacts to support
+having separate last name and first name to the checkout proccess in the frontend
```

Comment:
s/support/allow 
[...]the separation of first and last name during the check out process in the frontend.